src/coreclr/inc/icorjitinfoimpl_generated.h

```diff
@@ -417,7 +417,13 @@ CORINFO_ARG_LIST_HANDLE getArgNext(
 CorInfoTypeWithMod getArgType(
           CORINFO_SIG_INFO* sig,
           CORINFO_ARG_LIST_HANDLE args,
-          CORINFO_CLASS_HANDLE* vcTypeRet) override;
+          CORINFO_CLASS_HANDLE* vcTypeRet
+#if defined(TARGET_LOONGARCH64)
+          ,int *flags = NULL
+#endif
```

Comment:
Called by 
#62843 src/coreclr/jit/lclvars.cpp  Line-648,

for getting the arguments' info for calling-ABI.

But the interface between the JIT and EE has no function.
And the AMD-ABI add a platform-specific interface.

Reply:
AMD-ABI likes CEEInfo::getSystemVAmd64PassStructInRegisterDescriptor().

Maybe it's best to design a common interface for getting argType-info independent platform.

Reply:
This cannot be under ifdef.

It is ok to have architecture specific APIs like `getSystemVAmd64PassStructInRegisterDescriptor` on the JIT/EE interface. These APIs have to be present on all platforms, with dummy implementation on platforms that they are not relevant for.

I agree that having some sort of platform neutral interface would be nice, but it is hard to design it since there tends to be complex architecture-specific details around arg passing.

Reply:
(1) I will delete ifdef for new interface;
(2) designing a neutral interface is hard for all architecture, but at least for similar architecture is OK.

So, I think that the name of LoongArch64's interface should be neutral.